create_aio_app/main.py

```diff
@@ -27,11 +28,14 @@ def main():
     try:
         result = cookiecutter(template_path, **kwargs)
     except OutputDirExistsException as exc:
-        print('\n\n\033[91mDirectory with such name already exists!\033[00m\n')
+        print(click.style(
+            '\n\nDirectory with such name already exists!\n',
+            fg='red',
+        ))
         return
 
     folder = Path(result).name
 
-    print('\n\n\033[92mSuccessfully generated!\033[00m\n')
-    print(f'cd \33[94m{folder}/\033[00m')
+    print(click.style('\n\nSuccessfully generated!\n', fg='bright_green'))
```

Comment:
Same here, switch to click functions now and don't pollute stdout, it's meant for parsable data.

Reply:
Do you mean this message should also go to stderr ?

Reply:
Pretty much every non-parsable output should go stderr way as per POSIX conventions. That's how UNIX tooling can usually be easily combined into pipelines.

P.S. I'd insist on using click's helper functions instead of raw `print`s